### DISCUSSION TITLE
Overlapping Components

### DISCUSSION THREAD

**Discussion by jlsajfj** [charmbracelet/bubbletea#1216] [Q&A] [answered]
Looking at the website shows a bunch of windows, overlapping and being dragged- How was this achieved? Which example file corresponds?

**Comment by jlsajfj** [comment]
I'm trying to make overlays work... currently struggling!

**Comment by topi314** [comment, marked as answer]
see: https://github.com/charmbracelet/bubbletea/issues/169

**Reply by meowgorithm** [reply]
Yep, and also for something you can use immediately see: https://github.com/charmbracelet/lipgloss/pull/102

This is a feature in very active development right now in terms of official support, too.

**Reply by jlsajfj** [reply]
Thank you both!
